crates/wasmtime/src/runtime/coredump.rs

```diff
@@ -183,7 +183,10 @@ impl WasmCoreDump {
 
                         HeapType::Any => wasm_encoder::ValType::Ref(wasm_encoder::RefType {
                             nullable: true,
-                            heap_type: wasm_encoder::HeapType::Any,
+                            heap_type: wasm_encoder::HeapType::Abstract {
+                                ty: wasm_encoder::AbstractHeapType::Any,
+                                shared: false,
+                            },
                         }),
```

Comment:
While you're here mind updating this to the [`RefType::ANYREF`](https://docs.rs/wasm-encoder/latest/wasm_encoder/struct.RefType.html#associatedconstant.ANYREF) shortcut?

Reply:
(same for the changes below)